fastlane/Fastfile

```diff
@@ -22,8 +22,8 @@ end
 platform :ios do
 
   LANE_CONFIGS = [
-    LaneConfig.new(:test_iOS16_2, "Ably-iOS", ["iPhone 14 (16.2)"]),
-    LaneConfig.new(:test_tvOS16_1, "Ably-tvOS", ["Apple TV 4K (2nd generation) (16.1)"]),
+    LaneConfig.new(:test_iOS17_2, "Ably-iOS", ["iPhone 15"]),
```

Comment:
Why have you dropped the version numbers?

Reply:
I wanted to check what version it will select by default but somehow left it there. Sorry, fixed here - https://github.com/ably/ably-cocoa/pull/1851/commits/f443b48ae46d6dc0c2537d5ff6a87de7419a7d3b